Rollbar/RollbarQueueController.cs

```diff
@@ -212,17 +214,25 @@ private void ProcessQueues(AccessTokenQueuesMetadata tokenMetadata)
             }
         }
 
+        private readonly string[] criticalDataFields = new string[]
+        {
+            "access_token",
+        };
+
         private RollbarResponse Process(Payload payload, RollbarConfig config)
         {
             var client = new RollbarClient(config);
 
+            IEnumerable<string> safeScrubFields = 
+                JsonScrubber.FilterOutCriticalFields(config.ScrubFields, criticalDataFields);
```

Comment:
This is not what we want. We only want to keep `access_token` at the top level, i.e. the Rollbar `access_token`. If someone adds `access_token` to the list of scrub fields, we still want to scrub that if it occurs in the payload body.

Reply:
done.